machine/machine_core/machine_virtual.py

```diff
@@ -139,8 +149,8 @@ def stdchannel_redirected(stdchannel, dest_filename):
 
 
 class VirtNetwork:
-    def __init__(self, network=None, image="generic"):
-        self.locked = []
+    def __init__(self, network: Optional[int] = None, image: str = "generic"):
```

Comment:
Can we also rename `network` to `id` or so?

Reply:
Sure.